### PR TITLE
Reorganize CI files

### PR DESCRIPTION
**Pull request by benrubson**
Hi,

As discussed in #595, here's a PR which reorganize CI files, for easier upcoming updates.

Thank you 👍 

Ben

### CODE REVIEW

**Comment by benrubson** [comment]
_#583 should be merged first so that FreeBSD tests won't fail / timeout._

**Comment by benrubson** [comment]
This now builds / tests correctly, thus you should be able to approve it @rfjakob.
Many thx 👍

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+* text=auto eol=lf
```
Comment (rfjakob) [inline .gitattributes]: I think this should be in the Windows PR?

Comment (benrubson) [reply .gitattributes]: I've then removed it, I'll add it into Windows PR.
Thank you for your review 👍

**Comment by benrubson** [comment]
Let's approve this if you have some seconds @rfjakob ?
Many thanks 👍

**Pull request merged**